src/content/1/en/part1b.md

```diff
@@ -440,43 +440,43 @@ const referenceToAddition = arto.doAddition
 referenceToAddition(10, 15)   // 25 is printed
 ```
 
-Now the object has the method _doAddition_ which calculates the sum of numbers given to it as parameters. The method is called in the usual way using the object <em>arto.doAddition(1, 4)</em> or by storing a <i>method reference</i> in a variable and calling the method through the variable <em>referenceToAddition(10, 15)</em>.
+Now the object has the method _doAddition_ which calculates the sum of numbers given to it as parameters. The method is called in the usual way, using the object <em>arto.doAddition(1, 4)</em> or by storing a <i>method reference</i> in a variable and calling the method through the variable: <em>referenceToAddition(10, 15)</em>.
 
 If we try to do the same with the method _greet_ we run into an issue:
 
 ```js
-arto.greet()       // hello, my name is Arto Hellas gets printed
+arto.greet()       // "hello, my name is Arto Hellas" gets printed
 
 const referenceToGreet = arto.greet
-referenceToGreet() // prints only hello, my name is
+referenceToGreet() // prints "hello, my name is undefined"
```

Comment:
here in particular is the place where i think this concatenation in the example is advantageous. before, the comment is not really helpful in showing what happens.